recipe/meta.yaml

```diff
@@ -13,13 +13,11 @@ source:
     - windows_cmake_install_bindir.patch       # [win]
 
 build:
-  number: 0
-  # py36 since it uses vc14 like py35
-  skip: true  # [win and py36]
+  number: 1
   features:
     - vc9   # [win and py27]
     - vc10  # [win and py34]
-    - vc14  # [win and py35]
+    - vc14  # [win and (py35 or py36)]
```

Comment:
It is fine to have it here, but we only need to build for one of Python 3.5 or 3.6 as this is not a Python package and Python is only a proxy for VC, which both Python 3.5 and 3.6 have in common.